ui/components/src/StatChart.tsx

```diff
@@ -189,7 +191,7 @@ export function StatChart(props: StatChartProps) {
       },
       series: statSeries,
       textStyle: {
-        color: backgroundColor === 'transparent' ? '#000000' : '#FFFFFF',
+        color: backgroundColor === 'transparent' ? theme.palette.text.primary : '#FFFFFF',
```

Comment:
Looks like the linter caught a missing `theme` dependency here for the `useMemo`.